src/WebAdmin.cpp

```diff
@@ -341,6 +341,7 @@ void cWebAdmin::HandleWebadminRequest(cHTTPServerConnection & a_Connection, cHTT
 	if (ShouldWrapInTemplate)
 	{
 		cCSLock Lock(m_CS);
+		cLuaState::cLock lock(m_TemplateScript);
```

Comment:
Oh, I didn't even notice. You're right, that's poor style, I'll rename them both.